core/citrus-base/src/main/java/org/citrusframework/report/DefaultTestReporters.java

```diff
@@ -25,7 +25,7 @@
 public class DefaultTestReporters extends TestReporters {
 
     public static final List<TestReporter> DEFAULT_REPORTERS = Arrays.asList(
-            new LoggingReporter(),
+            new LoggingReporter(false),
```

Comment:
I'd rather use CitrusSettings in the LoggingReporter to obtain the default value for this setting.